<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionSave"/>
    <addaction name="actionOpen"/>
    <addaction name="actionClear"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCursor"/>
    <addaction name="actionNode"/>
    <addaction name="actionLine"/>
    <addaction name="actionDelete"/>
   </widget>
   <widget class="QMenu" name="menuCalculate">
    <property name="title">
     <string>Calculate</string>
    </property>
    <addaction name="actionCalculate"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
   <addaction name="menuCalculate"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionCursor"/>
   <addaction name="actionNode"/>
   <addaction name="actionLine"/>
   <addaction name="actionDelete"/>
   <addaction name="separator"/>
   <addaction name="actionCalculate"/>
   <addaction name="actionClear"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="rec.qrc">
     <normaloff>:/images/new.png</normaloff>:/images/new.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="rec.qrc">
     <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionNode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="rec.qrc">
     <normaloff>:/images/node.png</normaloff>:/images/node.png</iconset>
   </property>
   <property name="text">
    <string>Node</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionLine">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="rec.qrc">
     <normaloff>:/images/line.png</normaloff>:/images/line.png</iconset>
   </property>
   <property name="text">
    <string>Line</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionCursor">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="rec.qrc">
     <normaloff>:/images/cursor.png</normaloff>:/images/cursor.png</iconset>
   </property>
   <property name="text">
    <string>Cursor</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="actionCalculate">
   <property name="icon">
    <iconset resource="rec.qrc">
     <normaloff>:/images/calculate.png</normaloff>:/images/calculate.png</iconset>
   </property>
   <property name="text">
    <string>Calculate</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="rec.qrc">
     <normaloff>:/images/Clearup.png</normaloff>:/images/Clearup.png</iconset>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="rec.qrc">
     <normaloff>:/images/open.png</normaloff>:/images/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="rec.qrc">
     <normaloff>:/images/delete.png</normaloff>:/images/delete.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="rec.qrc"/>
 </resources>
 <connections/>
</ui>
